workalendar/america/barbados.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import absolute_import, division, print_function
+from datetime import date
+from dateutil import easter
+
+from ..core import WesternCalendar, ChristianMixin
+from ..core import Calendar, SUN, MON
+from ..registry import iso_register
+
+
+@iso_register("BB")
+class Barbados(ChristianMixin, WesternCalendar):
+    "Barbados"
+
+    include_good_friday = True
+    include_easter_sunday = True
+    include_easter_monday = True
+    include_whit_monday = True
+    include_boxing_day = True
+    shift_sunday_holidays = True
+    EASTER_METHOD = easter.EASTER_WESTERN
```

Comment:
since this class is a subclass of ``WesternCalendar`` you don't need this line.

Reply:
Removed.